src/promptflow/promptflow/tracing/_trace.py

```diff
@@ -228,7 +229,7 @@ def enrich_span_with_llm_model(span, output):
             from openai.types.completion import Completion
 
             if isinstance(output, (ChatCompletion, Completion)):
-                span.set_attribute("llm.model", output.model)
+                span.set_attribute("llm.response.model", output.model)
```

Comment:
is it better to be made as a constant too in the _constants.py?

Reply:
Thanks, good point.
We are refactoring the package structures and the _constants.py is also going to be moved.
We will move this definition as constant in future PRs, along with other attribute names, (e.g. "inputs", "output", etc).